### PR TITLE
Add mock kafka

### PR DESCRIPTION
**Pull request by gburanov**
Mock kafka is used both in dm and in bloom filter

### CODE REVIEW

File: topic_and_broker_test.go [hunk 1]
```diff
@@ -16,5 +16,5 @@ func TestSimpleTopicAndBrokerUsage(t *testing.T) {
 	require.True(t, subject.Available())
 	require.False(t, subject.IsEmpty())
 	require.Equal(t, subject.FirstBrokers(), "localhost:9092")
-	require.Equal(t, subject.FirstTopic(), "test")
+	require.Equal(t, len(subject), 2)
```
Comment (gburanov) [inline topic_and_broker_test.go:19]: the test was wrong, the topics and not strickly ordered...

File: mockkafka.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
+package sarama
+
+// TODO MOVE TO REX
```
Comment (itszootime) [inline mockkafka.go]: We can remove this

File: mockkafka.go [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+import (
+	"testing"
+	"time"
+)
+
+func Terminated(ms time.Duration, f func(), terminator func()) bool {
```
Comment (itszootime) [inline mockkafka.go:8]: This is not Kafka specific, is it used by multiple projects?

**Review by itszootime — COMMENTED** [review]
Left a few comments + `mock_kafka.go` is cleaner 🙂

**Comment by gburanov** [comment]
@itszootime  I wanted to do the same naming but we have mockbroker.go already =)

File: mockkafka.go (hunk 2 above)
Comment (gburanov) [reply mockkafka.go]: 👍

File: mockkafka.go (hunk 3 above)
Comment (gburanov) [reply mockkafka.go:8]: yes, it is also used by 2 projects (bloom and dm in exactly the same way). Maybe later we can move it out, but the fastest way will be to put it here as for now

File: mockkafka.go (hunk 2 above)
Comment (gburanov) [reply mockkafka.go]: done

**Review by itszootime — APPROVED** [review]
@gburanov ok, then this is consistent! 🙂

**Pull request merged**